apps/console.py

```diff
@@ -372,6 +379,9 @@ async def do_scan(self, params):
             else:
                 await self.device.start_scanning()
         elif params[0] == 'on':
+            if len(params) == 2:
```

Comment:
consider: parsing extra parameters as a sequence of `<name>=<value>`, with `filter=<filter>` for filtering, and `<filter>` would be `address:<pattern>`. This would make adding other arguments to `scan` easier later on.
ex: `scan on filter=address:01:02:.*`
(maybe overkill?)

Reply:
added!